backend/build_docs.py

```diff
@@ -0,0 +1,210 @@
+import os
+import subprocess
+import requests
+import sys
+import shutil
+import json
+from distutils.dir_util import copy_tree
+from shutil import copyfile
+
+source_dict: dict[str,str] = {}
+cur_dir: str = os.getcwd()
+
+
+def purge_folders(folder_paths: list[str]) -> None:
+    for folder_path in folder_paths:
+        if os.path.exists(folder_path):
+            shutil.rmtree(folder_path)
+            print(f'Purging, Folder {folder_path} deleted.')
+
+
+def update_source_dict(dir_path: str) -> None:
+    print(f'Updating source_dict for {dir_path}...')
+    for root, _, files in os.walk(dir_path):
+        for file in files:
+            file_path = os.path.join(root, file)
+            file_name = os.path.basename(file_path)
+            source_dict[file_path] = file_name
+
+
+def clone_repo(url: str, folder_name: str) -> None:
```

Comment:
Can you try to integrate the OR_docs and manpage together? At least tie them to the same commit hash number